Use Github Action to get update infomation from macchrome/macstable automatically

S0urceC0der opened the pull request:
Use Github Action to get update information from `macchrome/macstable` and update repo  automatically.

PLEASE Add Github token to project secrets.

S0urceC0der committed:
Create sync.yml

Commit:
update

actions-user committed:
Update tap

Commit:
update run

Commit:
Merge branch 'master' of github.com:S0urceC0der/homebrew-marmaduke-chromium

S0urceC0der committed:
Update sync.yml

update

Commit:
Update tap

actions-user committed:
Update tap

Commit:
change name

Commit:
Update command

S0urceC0der committed:
Update main.yml

auto test

actions-user committed:
Update tap

mtslzr commented:
Awesome! Wanted to do this for a little while, but hadn't gotten to it. Will give it a once-over and get it in place. Thanks!

actions-user committed:
Update tap

mtslzr commented on .github/workflows/main.yml:
```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Install Homebrew
         run: /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
       - name: Tap Repo
-        run: brew tap mtslzr/marmaduke-chromium
```
This should be swapped back before merging.

mtslzr commented:
Added a TOKEN with push access.

Also, mind adding the new version number into the README as well, during the automatic commit? Otherwise, should be good to go.

S0urceC0der committed:
Update sync.py

Update the version number in README, not sure the icon color has a specific meaning.

S0urceC0der committed:
Update main.yml

use the origin repo

S0urceC0der commented:
Update the version number in README, but not sure the icon color has a specific meaning.
